README.rst

```diff
@@ -19,7 +19,7 @@ kernel symbols from Windows and Linux.
 
 If you would like higher level semantic information, then we suggest using 
 LibVMI with PyVMI (python wrapper, included with LibVMI) and Volatility.
-Volatility (http://code.google.com/p/volatility/) is a forensic memory analysis
+Volatility (https://github.com/volatilityfoundation/volatility) is a forensic memory analysis
```

Comment:
Agreed on both.  We use each one in different ways.  Not sure if that's "a good thing", but it is where we are at today.